Realm/Realm.Sync/SyncConfiguration.cs

```diff
@@ -80,5 +81,17 @@ internal override Realm CreateRealm(RealmSchema schema)
             var srHandle = SharedRealmHandleExtensions.OpenWithSync(configuration, syncConfiguration, schema, EncryptionKey);
             return new Realm(srHandle, this, schema);
         }
+
+        internal override async Task<Realm> CreateRealmAsync(RealmSchema schema)
+        {
+            var syncConfiguration = new Native.SyncConfiguration
+            {
+                SyncUserHandle = User.Handle,
+                Url = ServerUri.ToString()
```

Comment:
We'll need to add ssl validation once https://github.com/realm/realm-dotnet/pull/1387 is merged